pkgs/applications/networking/cluster/helm/default.nix

```diff
@@ -2,16 +2,16 @@
 
 buildGoModule rec {
   pname = "kubernetes-helm";
-  version = "3.8.2";
+  version = "3.9.0";
   gitCommit = "6e3701edea09e5d55a8ca2aae03a68917630e91b";
```

Comment:
```suggestion
  gitCommit = "7ceeda6c585217a19a1131663d8cd1f7d641b2a7";
```

Reply:
Thanks for the reminder, I've now updated it.